taskcluster/scripts/build/android_build_debug.sh

```diff
@@ -12,8 +12,14 @@ git submodule update
 # translations
 ./scripts/utils/import_languages.py
 
-# $1 should be the qmake arch. 
-# Note this is different from what aqt expects as arch: 
+# Install Rust
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+export PATH="$HOME/.cargo/bin:$PATH"
+# Install Rust Android targets
+rustup target add x86_64-linux-android i686-linux-android armv7-linux-androideabi aarch64-linux-android
```

Comment:
Really just a nit, this can be reduced in the dockerfile - you can have this passed as an arg by taskcluster, so we only have the abi installed we need. 
Feel free to ignore, i clean that up later